File: FIlmFlix-Frontend/app/components/models/film.ts
import { Category } from '../enums/category';
export class Film {
 
    constructor(public filmTitle:string,
                public description: string, 
                public imageUrl:string, 
                public category: Category, 
                public voters:number,
                public resourceUrl:string,
                public published:string,
                public length:number) {}    
}

File: FilmFlix-Backend/src/FilmFlix/Models/Film.cs
using FilmFlix.Enums;
using System;
using System.ComponentModel.DataAnnotations;


namespace FilmFlix.Models
{
    public class Film
    {
        [Key]
        public int FilmId { get; set; }
        public string FilmTitle { get; set; }
        public string Description { get; set; }
        public string ImageUrl { get; set; }
        public Category Category { get; set; }
        public int Voters { get; set; }
        public string ResourceUrl { get; set; }
        public DateTime Published { get; set; } = DateTime.Now;
        public int Length { get; set; }
        
    }
}
